README.md

```diff
@@ -50,16 +50,36 @@ Use for:
 
 ## Getting Started with Local Tooling
 
+### Download Skaffold binary 
```

Comment:
I think we should keep this heading the same.  The top level README is really meant for users who just want to try out skaffold and are most likely not building from source.